Dockerfile-demo

```diff
@@ -0,0 +1,83 @@
+RUN apt-get install -y lsb-release gnupg2 software-properties-common \
+    && apt-get remove '.*gazebo.*' '.*sdformat.*' '.*ignition-math.*' '.*ignition-msgs.*' '.*ignition-transport.*' \
+    && sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list' \
+    && wget http://packages.osrfoundation.org/gazebo.key -O - | apt-key add - && apt-get update \
+    && wget https://bitbucket.org/osrf/release-tools/raw/default/jenkins-scripts/lib/dependencies_archive.sh -O /tmp/dependencies.sh \
+    && apt-get update
+
+# install Gazebo dependencies
+RUN /bin/bash -c "GAZEBO_MAJOR_VERSION=11 ROS_DISTRO=dummy source /tmp/dependencies.sh && echo \${BASE_DEPENDENCIES} \${GAZEBO_BASE_DEPENDENCIES} \${DART_DEPENDENCIES} | tr -d '\\' | xargs apt-get -y install" \
+    && apt-add-repository -y ppa:dartsim \
+    && apt-get install -y libdart6-dev
+
+# Build Dart and Gazebo
+RUN apt-get install -y mercurial \
+    && hg clone https://bitbucket.org/osrf/gazebo /tmp/gazebo \
+    && cd /tmp/gazebo \
+    && mkdir build && cd build  \
+    && cmake ../ \
+    && make -j4 \
+    && make install \
+    && rm -rf /tmp/gazebo
+
+# Finalize mesa installation, remove default ubuntu mesa libs
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y tzdata \
+    && ln -fs /usr/share/zoneinfo/Europe/Dublin /etc/localtime \
+    && dpkg-reconfigure -f noninteractive tzdata \
+    && apt-get install -y mesa-utils libopenmpi-dev python3-tk ssh gdb vim git \
+    && dpkg --force-depends -r libgl1 libglx0 va-driver-all libglx-mesa0 libgl1-mesa-dri
+
+ENV XVFB_WHD="1920x1080x24"\
+    DISPLAY=":0" \
+    LIBGL_ALWAYS_SOFTWARE="1" \
+    GALLIUM_DRIVER="llvmpipe" \
+    LP_NO_RAST="false" \
+    PKG_CONFIG_PATH="/usr/local/pkgconfig" \
+    LD_LIBRARY_PATH="/usr/local/lib:/usr/local/lib/python3.6/dist-packages/gymfc/envs/assets/gazebo/plugins/build/:/gymfc/examples/plugins/build/"
+
+# Build gymfc
+ADD setup.py gymfc.ini gen_pb.sh LICENSE.txt /gymfc/
+ADD gymfc /gymfc/gymfc
+RUN pip3 install /gymfc
+
+ADD tests /gymfc/tests
+ADD images /gymfc/images
+
+# Download solo model and gymfc plugins, build plugins
+RUN git clone https://github.com/wil3/gymfc-digitaltwin-solo.git /gymfc/demo \
+    && git clone https://github.com/wil3/gymfc-aircraft-plugins.git /gymfc/demo/plugins \
+    && sed -i "s/<plugin name='front_right_motor_model' filename='libgazebo_motor_model.so'>/<plugin name='config' filename='libAircraftConfigPlugin.so'> <motorCount>4<\/motorCount> <centerOfThrust> <link>base_link<\/link> <offset>0 0 0.058<\/offset> <\/centerOfThrust> <sensors> <sensor type='imu'> <enable_angular_velocity>true<\/enable_angular_velocity> <enable_linear_acceleration>true<\/enable_linear_acceleration> <enable_orientation>true<\/enable_orientation> <\/sensor> <\/sensors> <\/plugin><plugin name='front_right_motor_model' filename='libgazebo_motor_model.so'>/g;" /gymfc/demo/models/solo/model.sdf \
```

Comment:
What is this line doing?

Reply:
It adds the 'libAircraftConfigPlugin.so' plugin configuration to the solo model as this is mandatory for a model to run with GymFC as I learned from the respective error message

Reply:
As you mentioned elsewhere, I am very open to add this code directly to the solo plugin. That would make things easier to understand

Reply:
Yea I had a feeling thats what was going on. Havnt used the solo model in months so we can just migrate that change to that repo since it needs it anyway. I don't think the Solo has an ESC sensor so we can just enable the IMU meansurements. Ill have to do a little math to figure out its center of thrust is first.